go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/minio/operator
 
-go 1.22.0
-
-toolchain go1.22.4
+go 1.22.5
```

Comment:
```suggestion
go 1.22
```

Reply:
`go mod tidy`  is adding automatically the `.0` at the end, or maybe is just my IDE, but this seems fine to me, we just need to make sure is not added back again

Reply:
nope, not  building @harshavardhana